definition/data/sheets/ComplexTypes.json

```diff
@@ -252,5 +252,21 @@
   {"ID": "GeneralLetterContent", "ListElementCode": "letterContent", "ElementLabel": "Content for letter", "FieldType": "TextArea", "SecurityClassification": "Public"},
   {"ID": "GeneralLetterContent", "ListElementCode": "letterContentPrefix", "ElementLabel": "Yours sincerely,", "FieldType": "Label", "SecurityClassification": "Public"},
   {"ID": "GeneralLetterContent", "ListElementCode": "caseworkerName", "ElementLabel": " ", "FieldType": "Text", "SecurityClassification": "Public"},
-  {"ID": "GeneralLetterContent", "ListElementCode": "letterContentSuffix", "ElementLabel": "HM Courts & Tribunals Service", "FieldType": "Label", "SecurityClassification": "Public"}
+  {"ID": "GeneralLetterContent", "ListElementCode": "letterContentSuffix", "ElementLabel": "HM Courts & Tribunals Service", "FieldType": "Label", "SecurityClassification": "Public"},
+  {"ID": "ContactChangeContent", "ListElementCode": "contactChangesPrefix", "ElementLabel": "This will be sent to them as a letter or email.", "FieldType": "Label", "SecurityClassification": "Public"},
+  {"ID": "ContactChangeContent", "ListElementCode": "isTelephoneModified", "ElementLabel": "has Claimant phone changed?", "FieldType": "YesOrNo", "SecurityClassification": "Public"},
+  {"ID": "ContactChangeContent", "ListElementCode": "telephone", "ElementLabel": "Their phone number will be:", "FieldType": "Text", "SecurityClassification": "Public", "FieldShowCondition": "isTelephoneModified=\"Yes\""},
+  {"ID": "ContactChangeContent", "ListElementCode": "isEmailModified", "ElementLabel": "has Claimant email changed?", "FieldType": "YesOrNo", "SecurityClassification": "Public"},
```

Comment:
Do these questions not want to start with an upper case?

Reply:
done